diff --git a/node_modules/@bitnet-infotech/react-native-wav-to-mp3/android/src/main/java/com/wavtomp3/WavToMp3Package.kt b/node_modules/@bitnet-infotech/react-native-wav-to-mp3/android/src/main/java/com/wavtomp3/WavToMp3Package.kt
index 03d8af2..ab9caf8 100644
--- a/node_modules/@bitnet-infotech/react-native-wav-to-mp3/android/src/main/java/com/wavtomp3/WavToMp3Package.kt
+++ b/node_modules/@bitnet-infotech/react-native-wav-to-mp3/android/src/main/java/com/wavtomp3/WavToMp3Package.kt
@@ -4,35 +4,17 @@ import com.facebook.react.ReactPackage
 import com.facebook.react.bridge.NativeModule
 import com.facebook.react.bridge.ReactApplicationContext
 import com.facebook.react.uimanager.ViewManager
-import com.facebook.react.TurboReactPackage
-import com.facebook.react.module.model.ReactModuleInfo
-import com.facebook.react.module.model.ReactModuleInfoProvider
 
-class WavToMp3Package : TurboReactPackage() {
-    override fun getModule(name: String, reactContext: ReactApplicationContext): NativeModule? {
-        return when (name) {
-            WavToMp3Module.NAME -> WavToMp3Module(reactContext)
-            else -> null
-        }
+class WavToMp3Package : ReactPackage {
+    override fun createNativeModules(
+        reactContext: ReactApplicationContext,
+    ): List<NativeModule> {
+        return listOf(WavToMp3Module(reactContext))
     }
 
-    override fun getReactModuleInfoProvider(): ReactModuleInfoProvider {
-        return ReactModuleInfoProvider {
-            mapOf(
-                WavToMp3Module.NAME to ReactModuleInfo(
-                    WavToMp3Module.NAME,
-                    WavToMp3Module.NAME,
-                    false, // canOverrideExistingModule
-                    false, // needsEagerInit
-                    true, // hasConstants
-                    false, // isCxxModule
-                    true // isTurboModule
-                )
-            )
-        }
-    }
-
-    override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
+    override fun createViewManagers(
+        reactContext: ReactApplicationContext,
+    ): List<ViewManager<*, *>> {
         return emptyList()
     }
 }
